.github/workflows/ci-workflow.yml

```diff
@@ -135,4 +137,4 @@ jobs:
       - name: Run test
         env:
           CTEST_OUTPUT_ON_FAILURE: 1
-        run: pytest --tb=short -v tests/ --device ${{ matrix.model }}
+        run: PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python pytest --tb=short -v tests/ --device ${{ matrix.model }}
```

Comment:
Should not be necessary once https://github.com/LedgerHQ/ledgerctl/pull/51 is merged and deployed